### ISSUE TITLE
branch: phet-io-client-guides

### ISSUE THREAD

**Issue by chrisklus** [phetsims/chipper#955]
see https://github.com/phetsims/phet-io-client-guides/issues/2

needed because client guide generation is under development and so a phet-io build will fail under typical phet-io repo conditions. will be deleted once relevant work is complete and merged into master.

**Comment by zepumph** [comment]
Seems like we should delete this, right?

**Comment by samreid** [comment]
Yes, it looked all merged into main, so I deleted it. Closing.
